src/ComplaintCard.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import './ComplaintCard.less';
+
+export default class ComplaintCard extends React.Component {
+  render() {
+    const fields = [
+      'company',
```

Comment:
This would probably best be out of the render method, so we're not reallocating/creating garbage on every render (assuming the engine isn't hoisting the declaration).